src/middlewared/middlewared/service.py

```diff
@@ -870,6 +871,29 @@ def ping(self):
         "ping" protocol message.
         """
         return 'pong'
+    
+
+    def _ping_host(self, host, count, timeout):
+        process = run(['ping', '-c', f'{count}', '-W',  f'{timeout}', host])
```

Comment:
Can we please adjust the flags to reflect timeout in freebsd/debian ? I think you want `-t` for freebsd and `-w` for debian as uppercase `-W` is not a deadline.